vite-project-kabinet/src/constants/post.js

```diff
@@ -0,0 +1,8 @@
+export const post = {
```

Comment:
info: tohle nevidím použité. Nechtěla jsi to použít v `CardFullArticle`?

Reply:
Áno, chcela. Ale potom som tam čachrovala s tým id aby som ti urobila radosť a viac sa mi to tým pádom hodilo mať v tej kompoente :-D